test/functional/multiwallet.py

```diff
@@ -29,6 +30,10 @@ def run_test(self):
         os.mkdir(os.path.join(self.options.tmpdir, 'node0', 'regtest', 'w11'))
         self.assert_start_raises_init_error(0, ['-wallet=w11'], 'Error loading wallet w11. -wallet filename must be a regular file.')
 
+        # should not initialize if one wallet is a copy of another
+        shutil.copyfile(self.options.tmpdir + "/node0/regtest/w2", self.options.tmpdir + "/node0/regtest/w22")
```

Comment:
Use `os.path.join`?

Reply:
Done in e32c2a51b757ab764ec5dbe8bf04b3fc4aa3cb5f